doc/conf.py

```diff
@@ -101,11 +102,12 @@
     'navbar_pagenav': False,  # no "Page" navigation in sidebar
     'bootstrap_version': "3",
     'navbar_links': [
-        ("Examples", "auto_examples/index"),
-        ("API", "api"),
-        ("CLI", "generated/cli"),
-        ("What's new", "whats_new"),
-        ("GitHub", "https://github.com/mne-tools/mne-bids", True),
+        ("News", "whats_new"),
+        ("Install", "install"),
+        ("Use", "use"),
+        # ("API", "api"),
+        # ("CLI", "generated/cli"),
```

Comment:
the CLI docs do not seem to work: https://3251-89170358-gh.circle-artifacts.com/0/dev/generated/cli  --> clicked on a link here:
![grafik](https://user-images.githubusercontent.com/9084751/96505043-e0532e80-1255-11eb-8391-e96974914f39.png)

Reply:
Thank you, this is now fixed via 41c1cfd